uniffi_bindgen/src/bindings/gecko_js/templates/WebIDLTemplate.webidl

```diff
@@ -39,16 +42,16 @@ namespace {{ context.namespace()|class_name_webidl(context) }} {
 };
 {% endif -%}
 
-{%- for obj in ci.iter_object_definitions() %}
+{%- for obj in interfaces %}
```

Comment:
nit: consider naming the variable here `webidl_interfaces` or similar, by symmetry with the method names like `webidl_arguments`. I quite like the way they call out explicitly that we're working with the shadow types rather than the usual ones.